database/connector.py

```diff
@@ -246,13 +246,13 @@ def insert_to_events_data_collection(event_data: EventData):
     Returns:
         inserted_id
     """
-    event_data.machine_name = \
-        network_config["real_machine_identifier_name"]
+    event_data.machine_name = network_config["real_machine_identifier_name"]
```

Comment:
name should be changed here too right if we are changing above?

Reply:
right, fixed.